databricks/koalas/missing/indexes.py

```diff
@@ -30,7 +30,6 @@ def unsupported_property(property_name, deprecated=False, reason=""):
 class _MissingPandasLikeIndex(object):
 
     # Properties
-    T = unsupported_property('T')
```

Comment:
Should remove in `_MissingPandasLikeMultiIndex:` too

Reply:
thanks for the comment!